utils/framework/processlog-framework.psm1

```diff
@@ -52,9 +52,13 @@ function Invoke-ProcessLogs {
     } else {
         $job = $null
     }
+    $tempErrorActionPreference = $global:ErrorActionPreference
```

Comment:
Powershell uses `$ErrorActionPreference` for a variety of different behaviors; in docker builds, I have it set to "Stop" which is the strictest, but that also fails on a `Write-Error`. Since that's really not the behavior we want here, this overrides the error action preference just for within the processlog-framework to be "Continue", which is the default preference.